clay.h

```diff
@@ -415,7 +419,8 @@ Clay_FloatingElementConfig * Clay__StoreFloatingElementConfig(Clay_FloatingEleme
 Clay_CustomElementConfig * Clay__StoreCustomElementConfig(Clay_CustomElementConfig config);
 Clay_ScrollElementConfig * Clay__StoreScrollElementConfig(Clay_ScrollElementConfig config);
 Clay_BorderElementConfig * Clay__StoreBorderElementConfig(Clay_BorderElementConfig config);
-Clay_ElementId Clay__HashString(Clay_String key, uint32_t offset);
+Clay_ElementId Clay__HashString(Clay_String key, uint32_t offset, uint32_t seed);
+uint32_t Clay__GetParentId(void);
```

Comment:
We actually don't even need this function
```suggestion
uint32_t Clay__GetOpenLayoutElementId(void);
```